validator/Ballerina.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 org = "ballerinax"
 name = "health.fhir.r4.validator"
-version = "5.0.1"
+version = "5.0.2"
```

Comment:
version bump is done by the CD workflow. Do we have any specific reason to bump the version?